### PR TITLE
Update xcodeproj

### PR DESCRIPTION
**Pull request by pietbrauer**

### CODE REVIEW

**Comment by pietbrauer** [comment]
Not sure why `ROOT/bin/` is prefixed in all `execution_ouput.txt` files?

File: Gemfile.lock [hunk 1]
```diff
@@ -5,30 +5,30 @@ PATH
...
+    activesupport (3.2.22.1)
       i18n (~> 0.6, >= 0.6.4)
       multi_json (~> 1.0)
-    ast (2.0.0)
+    ast (2.2.0)
     bacon (1.2.0)
     claide (0.9.1)
     clamp (0.6.5)
-    clintegracon (0.5.2)
+    clintegracon (0.8.1)
```
Comment (mrackwitz) [inline Gemfile.lock]: @pietbrauer: This comes likely through the version bump here. I'll take care of that.

**Comment by pietbrauer** [comment]
@mrackwitz I reverted the update to `clintegracon` for now. I also updated the `Example.xcodeproj` `yml`-presentations, it seems to me as the `yml` files are generated using `xcodeproj` and these are valid changes.

**Comment by mrackwitz** [comment]
Whoops! Totally forgot about your PR sitting here. Sorry Piet!

>I also updated the `Example.xcodeproj` `yml`-presentations, it seems to me as the `yml` files are generated using `xcodeproj` and these are valid changes.

Not exactly. The YAML representations are [generated by the test harness](https://github.com/mrackwitz/xcres/blob/master/spec/spec_helper/xcodeproj_project_yaml.rb), but they use `xcodeproj`'s hash representations. The serialization, which has changed here, depends usually on the libyaml version in the environment. Not sure exactly how that has changed. But the changes seem valid. :+1:

**Pull request merged**